3 - generate a network of organisations using SRL/oecd_semantic_role_labelling.ipynb

```diff
@@ -0,0 +1,466 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    from allennlp_models import pretrained
</pre><p>allennlp and allennlp-models are missing as a dependency in <code>requirements.txt</code>!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/disaster-capitalism/named-entities/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
done